docs/src/main/sphinx/installation/containers.rst

```diff
@@ -1,9 +1,11 @@
-===================
-Trino in containers
-===================
+=============================
+Trino in the Docker container
+=============================
 
 The Trino project provides the `trinodb/trino <https://hub.docker.com/r/trinodb/trino>`_
-Docker image that includes the Trino server and a default configuration.
+Docker image that includes the Trino server and a default configuration and
+publishes it to Docker Hub. The image can be used in the Docker runtime, among
+several others.
```

Comment:
Dont forget about the info that its used by the Helm chart and link to k8s docs for that.